charts/openshift-metering/templates/reporting-operator/reporting-operator-config.yaml

```diff
@@ -34,7 +34,7 @@ data:
 {{- end }}
 {{- end }}
 
-  prometheus-url: {{ required "a valid reporting-operator.spec.config.prometheusURL must be set" $operatorValues.spec.config.prometheus.url | quote}}
```

Comment:
This was messing up the `make verify` tests, and we already do this validation in the ansible role, so I removed it.